### PR TITLE
Regd 165 body map

### PR DESCRIPTION
**Pull request by mingjiecn**
what I did in this ticket:
1 Create a TissueScoreBar component to display the color change according to the tissue score value. I can't find a free lib to create a linear gauge so I have to make one use react-chartjs-2 bar chart.
2. Update the way to use BodyMap for tissue score. Will show color for all the tissues, not just tissues exist for the type of data the view is showing. So I need to change the ways to get facet and enabled organ list.
3.  Only show BodyMap for GRCh38 since tissue score is the feature for GRCh38, not hg19.

### CODE REVIEW

File: components/tissue-score-bar.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  const organs = Object.keys(normalizedTissueSpecificScore);
+  let min = parseFloat(normalizedTissueSpecificScore[organs[0]][0]);
+  let max = parseFloat(normalizedTissueSpecificScore[organs[0]][0]);
+  organs.forEach((organ) => {
+    if (normalizedTissueSpecificScore[organ][1] === 1) {
+      min = parseFloat(normalizedTissueSpecificScore[organ][0]);
+    } else if (normalizedTissueSpecificScore[organ][1] === 10) {
+      max = parseFloat(normalizedTissueSpecificScore[organ][0]);
+    }
+  });
```
Comment (forresttanaka) [inline components/tissue-score-bar.js:35]: Is there any chance the minimum and maximum score could change someday? I feel like `MIN_SCORE` and `MAX_SCORE` would be more readable (it took me a while to understand why the numbers 1 and 10 are here), and — especially if needed throughout the code — would be a lot easier to change if you needed to in the future.

File: components/tissue-score-bar.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      data: [0.1 * i],
+      backgroundColor: colors[i],
+      barPercentage: 0.2,
+      borderColor: colors[i],
+      borderSkipped: false,
+      borderRadius: [
+        { topLeft: 0, topRight: 0, bottomLeft: 10, bottomRight: 10 },
+      ],
+    });
+  }
```
Comment (forresttanaka) [inline components/tissue-score-bar.js]: I think this maps `colors` to `datasets`, right? Would
```javascript
const datasets = TissueScoreHexColor.map((color, i) => …)
```
be simpler? Then if `TissueScoreHexColor` changes its array length, you don’t also have to manually update the `i <= 10`.

**Review by forresttanaka — APPROVED** [review]
Looks fine. I have just a couple suggestions.

File: components/tissue-score-bar.js (hunk 2 above)
Comment (mingjiecn) [reply components/tissue-score-bar.js]: @forresttanaka Thank you for the advice. Will update code accordingly!.

**Pull request merged**